docs/script.js

```diff
@@ -44,40 +44,130 @@ ReactDOM.render(<Component />, document.getElementById('react-1'));
 
 let Wrapper = React.createClass({
   getInitialState() {
-    return {interval: 2000};
+    return {
+      interval: 2000,
+      transitionTimeout: {
+        enter: 200,
+        leave: 200,
+      },
+    };
   },
   updateInterval(value) {
-    this.setState({interval: parseInt(value)});
+    this.setState({ interval: parseInt(value) });
+  },
+  updateTransitionTimeout(type, value) {
+    if(type === 'enter') {
+      this.setState({
+        transitionTimeout: {
+          enter: parseInt(value),
+          leave: this.state.transitionTimeout.leave,
+        }
+      });
+    }
```

Comment:
Can you make this one line and add a space after `if`? `} else if (type === 'leave') {`

Reply:
sure!